### ISSUE TITLE
Why is `se.metadata` capable of being `None`?

### ISSUE THREAD

**Issue by LTLA** [BiocPy/SummarizedExperiment#50]
```python
import summarizedexperiment as se
import numpy as np
out = se.SummarizedExperiment({ "counts": np.ndarray((10,20)) }
type(out.metadata)
## <class 'NoneType'>
```

This should just be a `{}`, so that downstreams can always assume `{}` and don't have to check against `None`.

**Comment by jkanche** [comment]
implemented in #52

**Comment by jkanche** [comment]
Fixed: #52
